spec/Section 4 -- Introspection.md

```diff
@@ -222,6 +217,25 @@ enum __DirectiveLocation {
 }
 ```
 
+### The __Schema Type
+
+The `__Schema` type is returned from the `__schema` meta-field and provides
+all information about the schema of a GraphQL service.
+
+Fields\:
```

Comment:
I've [checked this in Prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBidACAKgCzhgfQIGUw8BbAQ2wE8AHOAHSmd3wAMjSLL2MZ6+AJYBnDACc4MAK7iocACYYAZuIjl+eDJwIiycKn3JTKAWmVC4AGyWUoSumoBuQhXBHNKVqxiFRlEOJUMELQGJQARhDSMJr4ejwYEMrhGADi4pR0OACKADIYInDiLmBwAHTMzABiljYejEhVUABU2m564kJ0IdBGlDQSUrJQqcQwXVAA5kniGMBQ0t4AvpUsUCAANCAQPaFQIsiglOJqAO4ACicIhyheZwOH2xGZYADWUsR0lGB+U8gTaRwbYGCKKNwKPJ2KbSShTODVQLBELTZAgSgxCBbEA4GDkKwAdRwQngIm+ZWINxJQhcAjRYBETxAfiK4hgF0yUyoyGUXiK2wAViIAB4AIVeHxgxEoxjyfjgPL5wJAQuFxD+VjgOWkEHgiqs-JA33ErLRVHEbwUEDOG22jj8MAJrhgOGQAA4AAx2tRFAmZOhoxzuYpOBXbACOOvgHN2t3RIlM8kUimxkkjQkkHLh3KQvINyqK5CEAPEQO2Ig1WqjCtzSu2MEiToULuQACZ65khFY-gBhdQ5kDuACs2OkRSwkVuecNTiBAEl7AgpWAuj0AIL2cY0TX6orLZZAA) and the backslash does not get removed :+1: